webapp/package.json

```diff
@@ -19,7 +19,8 @@
     "clean": "npm run clean --workspaces --if-present"
   },
   "dependencies": {
-    "react-intl": "6.3.2"
+    "ajv": "8.12.0",
```

Comment:
why is this component needed? and is this component needed at the root level?

Reply:
I was also wondering the same. The problem is that I was encountering the following error (or something similar) while trying to install the dependencies:
`Error: Cannot find module 'ajv/dist/compile/codegen'`
Looking in the internet, the only way to address it that worked for me was to do just that, install the library. I haven't looked much deeper. Maybe it can be installed only as dev dependency, maybe only in the webapp, but I didn't dive much into it.

What do you suggest here?

Reply:
Do we know which library is requiring this? Or have we been using it the whole time and now a downstream package stopped requiring it? :P

Reply:
ajv is already required as of today by `schema-utils`, required by `webpack`. What I don't understand is why it was complaining for me, when it should be installed as a dependency :thinking:
Just for fun, I tried to removing it again and delete the node_modules folder and it seems not it is working fine, so... I will remove it.